packages/kit/src/exports/vite/index.js

```diff
@@ -789,3 +830,8 @@ export const files = [
 export const prerendered = [];
 export const version = ${JSON.stringify(config.kit.version.name)};
 `;
+
+/** @param {string} file */
+function read(file) {
```

Comment:
`filesystem.js` is probably a better place for this

Reply:
true!